lib/ReactViews/Guide/satellite-guidance.js

```diff
@@ -0,0 +1,38 @@
+export const guideData = [
+  {
+    imageSrc: require("../../../wwwroot/images/guides/satellite-feature.jpg"),
+    title: "Satellite Imagery: How do I…?",
+    body:
+      "A super-quick guide to getting the most out of historical satellite imagery.",
+    prevText: "No thanks",
+    nextText: "View guide"
+  },
+  {
+    imageSrc: require("../../../wwwroot/images/guides/satellite-zoom.jpg"),
+    title: "Where is my satellite imagery?",
+    body:
+      "You may need to adjust your zoom level to locate near real-time satellite imagery on the map.",
+    hidePrev: true
+  },
+  {
+    imageSrc: require("../../../wwwroot/images/guides/satellite-time.jpg"),
+    title: "Loading historical imagery",
+    body:
+      "In your data workbench, click on the capture date dropdown and select from available times to load historical imagery."
+  },
+  {
+    imageSrc: require("../../../wwwroot/images/guides/satellite-location.jpg"),
+    title: "Filtering imagery by location",
+    body:
+      "Satellite imagery may not always be available at your preferred time and location. If you only want to view imagery for a particular location, click ‘filter by location’ in your data workbench."
+  },
+  {
+    imageSrc: require("../../../wwwroot/images/guides/satellite-styles.jpg"),
+    title: "How do I view false-colour imagery?",
+    body:
+      "You can apply a range of false-colour styles to satellite imagery by clicking on the styles dropdown in your data workbench.",
```

Comment:
Suggest changing
"You can apply a range of false-colour styles to satellite imagery by clicking on the styles dropdown in your data workbench.",
to
"You can apply a range of styles, such as false-colour, to satellite imagery by clicking on the styles dropdown in your data workbench.",

Reply:
👍